infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/NodeDescriptor.java

```diff
@@ -22,26 +22,25 @@
     @Getter
     private final String clusterId;
 
-    // TODO: rename to connectionIdentifier (clean up nodeId and realNodeId) not changing to not break upper layers
     @Getter
-    private final UUID nodeId;            // Connection Identifier (APH UUID in the case of NSX, used by IClientChannelAdapter)
+    private final UUID connectionId;    // Connection Identifier (APH UUID in the case of NSX, used by IClientChannelAdapter)
 
     @Getter
-    private final UUID realNodeId;        // Represents the node's identifier as tracked by the Topology Provider
+    private final UUID realNodeId;      // Represents the node's identifier as tracked by the Topology Provider
```

Comment:
now that the other field was renamed, you can call this nodeId..

Reply:
Done